src/apps/components/AppPageTabs/AppPageTabs.tsx

```diff
@@ -0,0 +1,26 @@
+import { PageTab, PageTabs } from "@saleor/macaw-ui";
+import React, { ComponentProps } from "react";
+
+export enum AppPageTabValue {
+  THIRD_PARTY = "THIRD_PARTY",
+  WEBHOOKS_AND_EVENTS = "WEBHOOKS_AND_EVENTS",
+  SALEOR_APPS = "SALEOR_APPS",
+}
```

Comment:
We're trying to avoid new enums in the dashboard, especially if they don't come from the API. Can you use union type instead?

Reply:
FYI @andrzejewsky  ;)